tests/unit/email.spec.ts

```diff
@@ -0,0 +1,30 @@
+import Email from "@/utils/email";
+import { ValueError, ValidationError } from "@/errors";
+
+test("Email is requiring value", () => {
```

Comment:
Maybe "Email requires value" or "Email is not empty check"?

Reply:
Fixed